src/ocean/util/serialize/contiguous/Serializer.d

```diff
@@ -290,7 +290,7 @@ struct Serializer
                         // Recurse into static array elements which contain a
                         // dynamic array.
 
-                        foreach (element; s.tupleof[i])
```

Comment:
Ironically this loop has always been broken in two ways: Doing the `foreach` loop is wrong, and in the loop `element` should be passed, not `field`.